README.md

```diff
@@ -14,7 +14,7 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 ### Installation
```

Comment:
we can update the headers to be ##, ###, #### if needed to make section separation more obvious. I am not sure the three lines are a good solution. it looks a little heavy. if we decide to stick to 3 lines - we should add them before installation too. however I am not sold on that :/